ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh

```diff
@@ -91,6 +81,20 @@ source "${SHARED_DIR}/vsphere_context.sh"
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/govc.sh"
 
+# select a hardware version for testing
+vsphere_version=$(govc about -json | jq -r .About.Version | awk -F'.' '{print$1}')
```

Comment:
how about adding a whitespace between `print` and `$1`

Reply:
updated